xbzu1/1.0/board.xml

```diff
@@ -1,4 +1,5 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?> 
+<!DOCTYPE board SYSTEM "C:/Xilinx/Vivado/2021.1/data/boards/board_schemas/current/board.dtd">
```

Comment:
Agreed -- it's needed for the linter during debugging, so I left it in since I'm not done yet. We can either remove it or comment it out before final release of the files.